src/_includes/feature_lists/cloud.njk

```diff
@@ -25,6 +25,9 @@
         }, {
             label: "Fully Browser-Based",
             values: [null, 'check', 'check']
+        }, {
+            label: "Version History",
```

Comment:
```suggestion
            label: "Basic Version History",
```

Reply:
Better to manage expectations, especially as its in all editions leaving scope for a more advanced versioning feature (with or without Git) in future iterations

Reply:
No need to prepend basic in the marketing material I believe. On most features in the list we can prepend "Basic" as there's always room for improvement?